packages/firebase_admob/example/ios/Runner/AppDelegate.h

```diff
@@ -0,0 +1,6 @@
+#import <UIKit/UIKit.h>
```

Comment:
Sorry to have missed all of these  (since you'd pointed them out already). I think I've got them now.